core/src/main/scala/org/bitcoins/core/api/wallet/CoinSelector.scala

```diff
@@ -104,7 +104,7 @@ object CoinSelector extends CoinSelector {
       case None =>
         utxo.scriptWitnessOpt match {
           case Some(script) => script.bytes.length
-          case None         => 25 // PUBKEYHASH
+          case None         => 107 // PUBKEYHASH
```

Comment:
So they are wrong? https://github.com/bitcoinjs/coinselect/blob/becfd26096fd9fc719f5b4928935be61d87bbb98/utils.js#L6 

We should probably update they comment if they are